modules/virt-example-vlan-nncp.adoc

```diff
@@ -5,8 +5,14 @@
 [id="virt-example-vlan-nncp_{context}"]
 = Example: VLAN interface node network configuration policy
 
-Create a VLAN interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest
-to the cluster.
+Create a VLAN interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster.
+
+[NOTE]
+====
+Define all related configurations for a node's VLAN interface in a single `NodeNetworkConfigurationPolicy` manifest. For example, define the VLAN interface for a node and the related routes for the VLAN interface in the same `NodeNetworkConfigurationPolicy` manifest.
+
+When a node reboots, the NMState Operator cannot control the order in which policies are applied. Therefore, if you use separate policies for related network configurations, the NMState Operator might apply these policies in a sequence that results in a degraded network object.
```

Comment:
```suggestion
When a node restarts, the NMState Operator cannot control the order in which policies are applied. Therefore, if you use separate policies for related network configurations, the NMState Operator might apply these policies in a sequence that results in a degraded network object.
```

Reply:
1. [Usage of "reboot"](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r): Reboot is widely recognized among technical users, but where usage permits, use restart.
2. I searched "the NMState Operator" instances in the repo. The maximum no of instances were "the Kubernetes NMState Operator". Should we use "the Kubernetes NMState Operator" throughout the PR?